cypress/tests/components/dashboard/dashboard.spec.ts

```diff
@@ -0,0 +1,21 @@
+describe('admin dashboard', () => {
+  const DATA_FEDERATION = 'Data Federation';
+
+  before(() => {
+    cy.login();
+  });
+
+  after(() => {
+    cy.logout();
+  });
+
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it('admin dashboard should route properly once user clicks on operator tab under data services', function () {
+    cy.contains('Data Services').click();
+    cy.contains(DATA_FEDERATION).click();
+    cy.url().should('include', '/mcgms/cluster');
+  });
+});
```

Comment:
no need for separate folder

Reply:
let's not make a separate test for it...
let's make a common function similar to this https://github.com/red-hat-storage/odf-console/blob/master/cypress/views/odf-common.ts and call it in every card related test file...

Reply:
we have a `clickNavLink` selector as well...